spec/support/shared_vcr_setup.rb

```diff
@@ -27,7 +27,6 @@
 
   config.default_cassette_options = {
     record: :new_episodes,
-    re_record_interval: 2.months,
```

Comment:
@noremmie , this commit keeps the 2-month interval for specs (which I am still tempted to raise), but removes the re-recording requirement for cucumber tests. This seems safe, as the API calls in the cuke tests *should* all be covered by the specs, and it allows us to manage a smaller number of changes when re-recording is necessary.